utils/actix/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
```

Comment:
Is there a specific reason why crate name is different from the folder name? Same for the logger.

Reply:
It follows the current de-facto convention throughout the project. Discussed here: https://github.com/nearprotocol/nearcore/pull/2511#pullrequestreview-399223765